### PR TITLE
middle-ware wrapper

### PR DESCRIPTION
**Pull request by kamleshverma1**
- parse the response format as expected in UI.
- creating the data structure for inventory list

Signed-off-by: kamleshKumarVerma <kaverma@redhat.com>

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk @cloudbehl @anivargi @nthomas-redhat @shtripat @brainfunked Please review it .

Created the basic data structure for inventory list (cluster, host, file share, pool, rbd ) which going to be used for populating the data directly in UI after response parsed from middle-ware.
i am attaching UX design for all inventory list. so that you can easily identify what data we are going to populate in UI

1)  cluster list - https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/185937524
2) host list - https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/212034116
3) pool list - https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/198417118
4) rbd list - https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/198417119
5) file share list - https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/198417117

basically middle-ware part is in progress. ( i need one running server instance so that i can test the parsing stuff with real api response  ) and right now i have created the structure for inventory list according to UX design, you guys can review this for the time being.
